frontend/src/services/notebookService.ts

```diff
@@ -290,24 +290,32 @@ export const startNotebook = (data: StartNotebookData): Promise<Notebook> => {
   });
 };
 
-export const stopNotebook = (projectName: string, notebookName: string): Promise<Notebook> => {
+export const stopNotebook = async (
+  projectName: string,
+  notebookName: string,
+): Promise<Notebook> => {
   const dateStr = new Date().toISOString().replace(/\.\d{3}Z/i, 'Z');
-  const patch: RecursivePartial<Notebook> = {
+  const notebook = await getNotebook(projectName, notebookName);
+  const patch: Notebook = _.merge({}, notebook, {
     metadata: { annotations: { 'kubeflow-resource-stopped': dateStr } },
-  };
-
-  return patchNotebook(projectName, notebookName, patch);
+  });
+  return replaceNotebook(projectName, notebookName, patch);
```

Comment:
Why didn't we keep a patch here? We just want to stop, not rewrite it. I don't think we care at this time what other changes happened -- this is a legit patch.

When you bring it back, make delete just implement the `axios.patch` call just for it. No one else will need that.